qa/vagrant-helpers.rb

```diff
@@ -1,15 +1,56 @@
 # encoding: utf-8
 require "open3"
+require "bundler"
 
 module LogStash
+  class CommandExecutor
```

Comment:
are you talking about the main lib directory? or something inside the qa folder? my thoughts are more on keeping the qa framework all united inside the same directory, so later we could make it easy a single gem, what do you think?